pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -238,7 +232,16 @@ func (p *staticPolicy) Allocate(s state.State, pod *v1.Pod, container *v1.Contai
 		}
 		s.SetCPUSet(string(pod.UID), container.Name, cpuset)
 		p.updateCPUsToReuse(pod, container, cpuset)
+		// If the recently allocated CPU is for the last app container in the pod, release CPUs that are not re-used
+		// back to the default CPU set.
+		if container.Name == pod.Spec.Containers[len(pod.Spec.Containers)-1].Name{
+			cpusToReturn, ok := p.cpusToReuse[string(pod.UID)]
+			if  ok && ! cpusToReturn.IsEmpty(){
```

Comment:
Nit: few spacing issues. I suspect gofmt hasn't run?

```suggestion
			if ok && ! cpusToReturn.IsEmpty() {
```